### PR TITLE
coalib/__init__.py : Adding __version__ variable

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
I have added the __version__ variable to
check the version easier.

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
Referencing https://github.com/coala-analyzer/coala/issues/1662

**Comment by tushar-rishav** [comment]
@Adrianzatreanu Hey, just add ` Fixes: <full_url_for_issue >` in the end of your commit. That would essentially close the issue once the PR is merged. :)

**Comment by Adrianzatreanu** [comment]
@tushar-rishav  I did now :) thanks

File: coalib/__init__.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import sys
+import Constants
+
+__version__ = Constants.VERSION
```
Comment (tushar-rishav) [inline coalib/__init__.py]: Invalid import for `Constants`. Try ` from coalib.misc import Constants `

**Comment by Makman2** [comment]
I think a colon between `Fixes` and the issue is not allowed, see https://help.github.com/articles/closing-issues-via-commit-messages/ ^^

**Comment by Makman2** [comment]
Ah and while you are changing the commit message, you write `I changed [...]`, I'm not sure we the first person in commit bodies, I would suggest just writing `Adds the __version__ variable to [...]` :D
Not sure whether we have also guidelines for this inside our good-commit-message-guide ...

**Comment by Makman2** [comment]
unack 253cdb6

**Comment by AbdealiJK** [comment]
ack 	 1cf9d5d

**Comment by AbdealiJK** [comment]
Closing this and cherry picking it to https://github.com/coala-analyzer/coala/pull/1665 for easier merging

**Pull request closed without merging**